ignite/chainconfig/parser.go

```diff
@@ -108,3 +143,28 @@ func validateConfig(c *Config) error {
 
 	return nil
 }
+
+func validateNetworkConfig(c *Config) error {
+	if len(c.Validators) != 0 {
+		return &ValidationError{"no validators can be used in config for network genesis"}
+	}
+
+	for _, account := range c.Accounts {
+		// must have valid bech32 addr
+		if _, _, err := bech32.DecodeAndConvert(account.Address); err != nil {
+			return fmt.Errorf("invalid address %s: %w", account.Address, err)
+		}
+
+		if account.Coins == nil {
+			return &ValidationError{"account coins is required"}
+		}
+
+		if account.Mnemonic != "" {
+			return &ValidationError{"cannot include mnemonic in network config genesis"}
+		}
+	}
+
+	// TODO: We should validate all of the required config fields
```

Comment:
Couldn't be done trivially by defining a `validateBaseConfig`, having common checks for both config types?

Reply:
The main thing is that the checks are different - even for the `BaseConfig` fields.  This TODO is mainly there for validating Genesis fields, but I'm not sure how we can do that from within this function.  We call "ValidateGenesis" later so its probably fine.

Reply:
Ok, I misunderstand

We perform `ValidateGenesis` and `InitGenesis` with node start simulation, therefore this is fine we can remove this TODO.
If we keep a TODO, we can create an issue to track it

Reply:
I think it's ok to remove TODO in this case since `ValidateGenesis` is called by the node eventually